pkg/services/accesscontrol/seeder/roles.go

```diff
@@ -0,0 +1,68 @@
+package seeder
+
+import "github.com/grafana/grafana/pkg/services/accesscontrol"
+
+var builtInRoles = []accesscontrol.RoleDTO{
+	{
+		Name:    "grafana:builtin:users:read:self",
+		Version: 1,
+		Permissions: []accesscontrol.Permission{
+			{
+				Permission: "users:read",
+				Scope:      "users:self",
+			},
+			{
+				Permission: "users.tokens:list",
+				Scope:      "users:self",
+			},
+			{
+				Permission: "users.teams:read",
+				Scope:      "users:self",
+			},
+		},
+	},
+	{
+		Name:    "roles:adminUsers:viewer",
+		Version: 1,
+		Permissions: []accesscontrol.Permission{
+			{
+				Permission: "users.authtoken:list",
+				Scope:      "*",
+			},
+		},
+	},
+	{
+		Name:    "roles:adminUsers:editor",
+		Version: 1,
+		Permissions: []accesscontrol.Permission{
+			{
+				Permission: "users.password.update",
+				Scope:      "*",
```

Comment:
It does not seem that evaluator supports `*` now. And currently we are forwarding `users:[id]` with the request and the evaluator is going to check against that specific scope. 

That said, probably I need to add support of `*` in the evaluator.

Reply:
Ah I might be wrong actually, it does seem evaluator does this
```
rule, err := glob.Compile(dbScope, ':', '/')
			if err != nil {
				return false, err
			}

			match = rule.Match(s)
			if match {
				break
			}
```